php/Terminus/Site.php

```diff
@@ -121,9 +121,13 @@ public function availableEnvironments() {
    */
   public function info($key = null) {
     $info = $this->information;
-    if ($key AND property_exists($info, $key)) {
+    if ($key && property_exists($info, $key)) {
       return $info->$key;
     }
+    if ($key && !property_exists($info, $key)) {
+      // Asking for info on a key that is not found.
+      return null;
+    }
```

Comment:
You could cull two lines rom this:
`public function info($key = null) {
  $info = $this->information;
  if ($key) {
    return property_exists($info, $key) ? $info->$key : null;
  }
  return $info;
}`